ngraph/src/ngraph/builder/split.hpp

```diff
@@ -49,6 +49,7 @@ namespace ngraph
         ///
         /// \return     The vector containing multiple outputs we split input node into.
         ///
+        NGRAPH_API
```

Comment:
why we expose this builders? I think we always suggest to use operations from opsets

Reply:
It uses in group_convolution which I moved to tests.
I have a plan to remove this builders for v0 ops.